app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<h2>Tasks</h2>
+<ul >
+  <% @tasks.each do |task| %>
+  <li>
+
+    <section class="line">
+
+      <% if task.completion_date.to_s <= DateTime.now.strftime("%Y-%m-%d")%>
+        <div class="strike"><strong>
+          Action: <%= task.action.capitalize %>
+        </strong> </div>
+        <%= button_to "(Unmark complete)", complete_task_path(task.id), method: :patch %>
+      <% else %>
+        <strong>
+          Action: <%= task.action.capitalize %>
+        </strong>
+        <%= button_to "(Mark complete)", complete_task_path(task.id), method: :patch %>
+      <% end %>
+
+      <%= link_to "(View details)", task_path(task.id) %>
+      <%= link_to "(Delete)", task_path(task.id), method: :delete, data: { confirm: "You sure?" }%>
+    </section>
+  </li>
+  <% end %>
+
+  <div>
+    <%= link_to "(Add new book)", new_task_path %>
+  </div>
```

Comment:
Technically this `div` shouldn't be nested inside of the `ul`! It should be outside of it.